portal-kernel/src/com/liferay/portal/kernel/theme/ThemeDisplay.java

```diff
@@ -2018,6 +1990,39 @@ private int _getLayoutManagePagesInitialChildren() {
 		return _layoutManagePagesInitialChildren;
 	}
 
+	private String _getResource(
+		String staticResourceURLName, String staticResourceURLPath,
+		String unhashedFileURIName, String unhashedFileURIPath) {
+
+		String url;
+
+		String prefix = PortalUtil.getPathModule();
```

Comment:
why not `pathModule` instead of `prefix`?`

Reply:
oh yes, so you `portal.getPathModule()` may contain `context` + ' /o` so you are forced to substract the `proxy`.

So, this is easier than building the URL by concatenating. And we have no name for this piece of the url, as it's not the original `pathModule`, neither `Portal.PATH_MODULE`. But instead is something like `pathModuleMinusProxy` or `contextAndModulePath` .  Such names are ugly anyways so I guess `prefix` is fine